hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/CompactSplit.java

```diff
@@ -218,14 +221,14 @@ public synchronized boolean requestSplit(final Region r) {
     return false;
   }
 
-  public synchronized void requestSplit(final Region r, byte[] midKey) {
+  private synchronized void requestSplit(final Region r, byte[] midKey) {
```

Comment:
What is the reason to change the visibility of this method?

Reply:
Thanks for reviewing, @apurtell. This method and the method mentioned in the next conversation are only used inner the class, so I changed them to be private.